app/models/manageiq/providers/autosde/storage_manager.rb

```diff
@@ -224,13 +224,14 @@ def self.params_for_create
               :validationDependencies => %w[type endpoints.default.hostname authentications.default.userid authentications.default.password],
               :fields                 => [
                 {
-                  :component  => "select",
-                  :id         => "endpoints.default.security_protocol",
-                  :name       => "endpoints.default.security_protocol",
-                  :label      => _("Security Protocol"),
-                  :isRequired => true,
-                  :validate   => [{:type => "required"}],
-                  :options    => [
+                  :component    => "select",
+                  :id           => "endpoints.default.security_protocol",
+                  :name         => "endpoints.default.security_protocol",
+                  :label        => _("Security Protocol"),
+                  :isRequired   => true,
+                  :validate     => [{:type => "required"}],
+                  :initialValue => 'ssl-no-validation',
```

Comment:
@gregoryb-at-ibm it seems like you don't want an initial value here based on https://github.com/ManageIQ/manageiq-providers-autosde/pull/59 is that right?

Reply:
I actually created #59 because the initial value stopped working. I tried fixing it by adding `initialValue='ssl-no-validation'` like in this PR. But that didn't fix the issue. The form still wouldn't validate for me.

I decided that since we are going to add 2 more options soon anyway, and when we do we won't want a default, it's OK to just remove the default now.

We can try working together to try and figure out why it's not working for me if you want.

Reply:
Here's how it looks like when I use `initialValue`

![image](https://user-images.githubusercontent.com/68283004/108225864-01b12200-7145-11eb-9e39-f9aacdeb9e96.png)

![image](https://user-images.githubusercontent.com/68283004/108225910-0e357a80-7145-11eb-9c91-57150b7abbaa.png)

Reply:
@Fryguy 
FYI ^^
This might mean there's a bug in how `initialValue` works.

Reply:
This is actually a bug in Carbon itself, it implicitly selects a field but doesn't tell the validator. According to the DDF people: "no one can implement a dropdown properly" :laughing: